vignettes/p2visualisation.Rmd

```diff
@@ -0,0 +1,232 @@
+  geom_node_text(aes(label=name), family = "serif", size = 2.5) +
+  scale_edge_width(range = c(0.3,1.5)) +
+  theme_graph() +
+  theme(legend.position = "none")
+```
+
+![plot of chunk ggrapheg](figure/ggrapheg-1.png)
+  
+As we can see in the code above, we can specify various aspects of the plot to 
+tailor it to our network. 
+Firstly, we can alter the **layout** of the network using the `layout =` argument
+to create a clearer visualisation of the ties between nodes. 
+This is especially important for larger networks, where nodes and ties are more 
+easily obscured or misrepresented.
+In `{ggraph}`, the default layout is the "stress" layout. 
+The "stress" layout is a safe choice because it is deterministic and 
+fits well with almost any graph, but it is also a good idea to explore and try 
+out other layouts on your data.
+More layouts can be found in the `{graphlayouts}` and `{igraph}` R packages. 
+To use a layout from the `{igraph}` package, enter only the last part of the layout 
+algorithm name (eg. `layout = "mds"` for "layout_with_mds").
+  
+Secondly, using `geom_node_point()` which draws the nodes as geometric shapes 
+(circles, squares, or triangles), we can specify the presentation of **nodes** 
+in the network in terms of their *shape* (`shape=`, choose from 1 to 21), 
+*size* (`size=`), or *colour* (`colour=`). We can also use `aes()` to match to 
+node attributes. To add labels, use `geom_node_text()` or 
+`geom_node_label()` (draws labels within a box). The font (`family=`), 
+font size (`size=`), and colour (`colour=`) of the labels can be specified.
+  
+Thirdly, we can also specify the presentation of **edges** in the network.
+To draw edges, we use `geom_edge_link0()` or `geom_edge_link()`. 
+Using the latter function makes it possible to draw a straight line with a 
+gradient. 
+The following features can be tailored either globally or matched to specific
+edge attributes using `aes()`:
+  
+* *colour*: `edge_colour=`
+  
+* *width*: `edge_width=`
+  
+* *linetype*: `edge_linetype=`
+  
+* *opacity*: `edge_alpha=`
+  
+For directed graphs, arrows can be drawn using the `arrow=` argument and the 
+`arrow()` function from `{ggplot2}`. The angle, length, arrowhead type, and
+padding between the arrowhead and the node can also be specified.
+  
+To change the position of the legend, add the `theme()` function from `{ggplot2}`.
+The legend can be positioned at the top, bottom, left, or right, 
+or removed using "none".
+
+For more see David Schoch's [excellent resources on this](http://mr.schochastics.net/netVizR.html).
+  
+# Exporting plots to PDF
+
+We can print the plots we have made to PDF by point-and-click
+by selecting 'Save as PDF...' from under the 'Export' dropdown menu
+in the plots panel tab of RStudio.
+
+If you want to do this programmatically,
+say because you want to record how you have saved it so that you can
+e.g. make some changes to the parameters at some point,
+this is also not too difficult.
+
```

Comment:
Suggestion: remove lines 214-223 in plot. Keep only last paragraph about saving and removing point and click operations.

Reply:
Suggestion:  "For saving your (gg-based) plot programmatically," (line 225)